site/templates/collection/slider.jet

```diff
@@ -3,8 +3,8 @@
 {{block pageCollectionSlider()}}
 {{if len(.Items) > 0}}
 <section class="page-collection page-collection-slider page-collection-slider-{{.ItemsPerRow}} page-collection-slider-{{.ItemLayout}} slider-{{.ItemsPerRow}} slider-{{.ItemLayout}} page-collection-{{.ItemsPerRow}}-slider" aria-label="{{i18n("wcag_aria_label_collection_slider")}}">
-  <div class="swiper-wrapper-container">
-    <div class="swiper-container" data-items-per-row="{{.ItemsPerRow}}" data-layout="{{.ItemLayout}}">
+  <div class="collection-wrapper-container swiper-wrapper-container">
+    <div class="collection-container swiper-container" data-items-per-row="{{.ItemsPerRow}}" data-layout="{{.ItemLayout}}">
```

Comment:
Haven't we removed the `swiper-wrapper-container` and `swiper-container` css classes or are they still required for swiper.js?

Reply:
Only the `swiper-container` class is required for swiper.js. It occurs [once in the swiper.min.js]( https://github.com/shift72/core-template/blob/develop/site/static/scripts/swiper.min.js) file.

The main reason we still have to keep these classes is for shared templates as they both appear in [main.js](https://github.com/shift72/core-template/blob/develop/site/static/js/main.js) also